src/WebJobs.Script/Diagnostics/ConditionalTraceWriter.cs

```diff
@@ -38,33 +36,6 @@ public override void Trace(TraceEvent traceEvent)
             {
                 InnerWriter.Trace(traceEvent);
             }
-            else if (_failedConditionBehavior == FailedConditionTraceBehavior.TraceAsVerbose)
```

Comment:
I removed all this because I think it was complicating things. We'll be able to use our system traces soon to capture ALL trace output, and that should be sufficient for debugging scenarios (i.e. they don't need to go to disk).